src/ol/renderer/canvas/vectortilelayer.js

```diff
@@ -276,7 +276,7 @@ ol.renderer.canvas.VectorTileLayer.prototype.createReplayGroup = function(tile,
 /**
  * @inheritDoc
  */
-ol.renderer.canvas.VectorTileLayer.prototype.forEachFeatureAtCoordinate = function(coordinate, frameState, callback, thisArg) {
+ol.renderer.canvas.VectorTileLayer.prototype.forEachFeatureAtCoordinate = function(coordinate, frameState, hitTolerance, callback, thisArg) {
   var resolution = frameState.viewState.resolution;
   var rotation = frameState.viewState.rotation;
   var layer = this.getLayer();
```

Comment:
As i don't know how to setup a test for this, i can't really work on this problem.
How could i set up a vectortilelayer containing only one feature which is very near to a corner of tile?
i tried something like this, but it does not seem to work: 

      // 1 x 1 pixel black icon
      var img = document.createElement('img');
      img.onload = function() {
        var target = document.createElement('div');
        target.style.width = '256px';
        target.style.height = '256px';
        document.body.appendChild(target);

        var tileGrid = ol.tilegrid.createXYZ();
        var zoom = 10;
        //choose coordinate at an edge of the tileGrid
        var coordinate = ol.extent.getTopLeft(tileGrid.getTileCoordExtent([zoom, 0, 0]));

        var map = new ol.Map({
          view: new ol.View({
            center: coordinate,
            zoom: zoom
          }),
          target: target
        });

        var feature = new ol.Feature(new ol.geom.Point(coordinate));
        feature.setStyle([new ol.style.Style({
          image: new ol.style.Icon({
            img : img,
            imgSize: [1, 1]
          })
        })]);

        var TileClass = function() {
          ol.VectorTile.apply(this, arguments);
          if (this.tileCoord[0] === zoom && this.tileCoord[1] === 0 && this.tileCoord[2] === 0) {
            this.setFeatures([feature]);
          }
          this.setState('loaded');
          this.setProjection(ol.proj.get('EPSG:3857'));
        };
        ol.inherits(TileClass, ol.VectorTile);

        var source = new ol.source.VectorTile({
          format: new ol.format.MVT(),
          tileClass: TileClass,
          tileGrid: tileGrid
        });
        var layer = new ol.layer.VectorTile({
          source: source
        });
        map.addLayer(layer);
        map.renderSync();

        var cb0 = sinon.spy();

        var pixel = map.getPixelFromCoordinate(coordinate);

        map.forEachFeatureAtPixel(pixel, cb0);
        expect(cb0.callCount).to.be(1);  // this already fails
        expect(cb0.firstCall.args[1]).to.be(layer);

Another question about the vectortilelayer for me is: how are features stored which are in multiple tiles? Could a simple buffering up the extent result in multiple calls of the callback for the same feature?